test/extended/util/annotate/rules.go

```diff
@@ -300,6 +300,9 @@ var (
 			`\[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] Granular Checks: Services Secondary IP Family should function for client IP based session affinity: http`,
 			`\[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] Granular Checks: Services Secondary IP Family should function for client IP based session affinity: udp`,
 			`\[sig-network\] \[Feature:IPv6DualStack\] \[LinuxOnly\] should have ipv4 and ipv6 node podCIDRs`,
+			`\[sig-network\] \[Feature:IPv6DualStack\] Granular Checks: Services Secondary IP Family \[LinuxOnly\] should function for client IP based session affinity: http`,
```

Comment:
the bug referenced for this skip set is verified and the jira referenced for this block has its PR merged.  Why are we still skipping these?

Reply:
Thanks @deads2k. I think, referenced PR in jira has been merged in upstream https://github.com/ovn-org/ovn-kubernetes/pull/2294 but not in downstream https://github.com/openshift/ovn-kubernetes/pull/666.

Since service affinity is not supported, bug was verified after skipping these three tests for OVN-K.

Reply:
service affinity should work once 666 merges. The only thing that isnt supported is:
https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-kind.sh#L30